src/facade/conn_context.h

```diff
@@ -65,6 +65,8 @@ class ConnectionContext {
   dfly::acl::AclKeys keys{{}, true};
   // pub/sub
   dfly::acl::AclPubSub pub_sub{{}, true};
+  // db index, empty optional means ALL db's
```

Comment:
should this comment be update ? empty optional? 
acl_db_idx should have default 0 ?

Reply:
I updated the comment and added the default (although is not really needed since we get the value from the user we are authed with))